app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./controllers/topicsController');
+const db = require('./db'); // Import the database pool
+
+app.use(express.json());
```

Comment:
We shouldn't write code we aren't using yet - there's no need for this to be here, we really should be setting up our repo with the things we need, not the things we *think* we might need later down the line. Generally just good practice to work towards the specs of the tickets we are given instead of going on our own adventures :)